server/models/serverResponseModel.ts

```diff
@@ -0,0 +1,9 @@
+import { Types } from 'mongoose'
```

Comment:
this is not a mongoose model, so place it somewhere related to graphql

Reply:
Moved.